cat/CaT-service.yaml

```diff
@@ -3296,7 +3296,32 @@ components:
               type: string
               format: date-time
               description: Date/time - Event Created
-
+            buyerQuestions:
+              type: array
+              items:
+                allOf:
+                  - $ref: '#/components/schemas/EvalCriteria'
+                  - type: object
+                    properties:
+                      relatesTo:
+                        type: string
+                      relateItems:
+                        type: string
+                      requirementGroups:
```

Comment:
Might this be better called `questionGroups`? (as RequirementGroup is a different schema in the CCS-OCDS schemas - could be a bit confusing?)

Reply:
Should this be an array? (multiple groups?)

Reply:
Definitely should be an array. Just tryig to work out if covered by the OCDS constraint for the name